src/util/response.js

```diff
@@ -0,0 +1,39 @@
+const logger = require('../lib/logger/bunyan');
+
+class SendResponse {
+
+    errorMsg(req, res, status, message, error, scope) {
+        console.log(error);
+        logger.logResponse(req.id, {
+            status: status,
+            message: message
+        }, 200);
+        res.status(status).json({
+            message: message
+        });
+    };
+
+    successMsg(req, res, status, data, message) {
+        logger.logResponse(req.id, {
+            status: status,
+            message: message,
+            data: data
+        }, 200);
+        res.status(status).json({
+            message: message,
+            data: data
+        })
+    };
+
+    customMsg(req, res, status, message) {
+        logger.logResponse(req.id, {
+            status: status,
+            message: message
+        }, 200);
+        res.status(status).json({
+            message: message
+        })
+    };
+};
+
+module.exports = new SendResponse();
```

Comment:
Hey @whyDontI,

Can you add the info mation about using logging levels and SendResponse in the Contribution.md file that will be great

Reply:
Sure